contrib/filter-repo-demos/git_lint_history.py

```diff
@@ -91,13 +96,17 @@ def is_relevant(filename):
         help=("Python code for determining whether to apply linter to a "
               "given filename.  Implies --filenames-important.  See CALLBACK "
               "below."))
+parser.add_argument('--relevant-glob', metavar="GLOB", action='append',
+        help=("Glob of paths to include in filtered history. Can be specified "
+              "multiple times to get a union of paths."))
```

Comment:
The --relevant-glob and the forwarding of filter-repo options appear to be two separate things that probably belong in separate commits.

Also, the description talks about whether to include paths in history, but --path-glob already exists for that and since you're now forwarding git-filter-repo options it would seem to make sense to use it.  But is there a description error here?  Later, the code changes you make seem to suggest you are not using it to filter which blobs are included in history, but just using it to filter which blobs get linted while cleaning history while the others are left alone.